A14-channelz.md

```diff
@@ -29,19 +29,23 @@ and connections that process currently has.  In a process, there are multiple
 clients and servers.  To avoid confusion, a strict terminology will be used:
 
 1.  A "channel" represents an abstraction that can start and complete RPCs.  A
-    channel may have subchannels which are themselves also channels.  A
-    channel is a directed acyclic graph (DAG).  Only leaf channels may have
-    sockets.  Interior channels may only have subchannels.
-2.  A "server" represents the entry point for RPCs.  A server may have one or
-    more listening sockets, and has a collection of "services".  Unlike clients,
-    servers are not hierarchical.  A server may only have sockets.
-3.  An "endpoint" is either a channel or a server.  The local and remote
+    channel may have channels and subchannels.  A channel is a directed 
+    acyclic graph (DAG).  Only leaf channels may have
+    sockets.  Interior channels may only have channels and subchannels.
+2.  A "subchannel" represents an abstraction that is load balanced over by an 
```

Comment:
Suggest saying something like this, maybe right after the first sentence here: "A subchannel will only send RPCs that are delegated to it by a parent channel or subchannel."

Reply:
Done.